lib/cache.ts

```diff
@@ -0,0 +1,28 @@
+const cache: {
+  [key: string]: {
+    value: string;
+    ts: number;
```

Comment:
maybe something like expire_at is better

Reply:
Good point :joy: